plotter/templates/profile_table.pt

```diff
@@ -3,9 +3,18 @@ If the profile name link is not active for a recently-uploaded profile,
 you need to wait a few minutes for the server to process the
 data. </p>
 
+  <form method="get" action="/trackhub_list/${user}/">
+    <button type="submit">View Your Track Hubs</button>
+</form>
+
+<form method="get" action="/trackhub_init/${user}/">
+    <button type="submit">Create Track Hub</button>
```

Comment:
sorry for not mentioning this earlier but it is usually a bad idea to use a form if there are no fields/data to submit, as is the case above. in this case you could just use a standard link, `a href` instead, right?

and actualy i would suggest changing the URL scheme from /trackhub_*/${user}/ to simply /trackhub_*/ because anyways the user variable is stored in the session info